Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM 300288021642.dkr.ecr.eu-west-2.amazonaws.com/ch-serverjre:1.2.5
+FROM 300288021642.dkr.ecr.eu-west-2.amazonaws.com/ch-serverjre:1.2.5 AS builder
+
+# The ARGS are not retained in the final image, due to using a multi-stage build.
+ARG ARTIFACTORY_URL
+ARG ARTIFACTORY_USERNAME
+ARG ARTIFACTORY_PASSWORD
 
 ENV ORACLE_HOME=/apps/oracle \
-    ARTIFACTORY_BASE_URL=https://artifactory.companieshouse.gov.uk/artifactory/virtual-release
+    ARTIFACTORY_BASE_URL=${ARTIFACTORY_URL}/virtual-release
 
 RUN yum -y install gettext && \
```

Comment:
These binaries are being installed into the builder, but aren't they needed in the final image, so need to be moved to the 2nd stage?